srcs/runners_manager/vm_creation/openstack/OpenstackManager.py

```diff
@@ -23,17 +23,20 @@ class OpenstackManager(CloudManager):
     """
     Manager related to Openstack virtual machines
     """
+    SETTING_OBJ = OpenstackConfig
```

Comment:
Did I miss something or this needs to be renamed?
```suggestion
    CONFIG_SCHEMA = OpenstackConfig
```